test/RouterUniV3.js

```diff
@@ -2,9 +2,12 @@ const { ethers } = require('hardhat');
 const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
 const { ether, constants } = require('@1inch/solidity-utils');
 const { ProtocolKey } = require('./helpers/utils');
-const { initRouterContracts } = require('./helpers/fixtures');
+const { initRouterContracts, encodePathExactInput } = require('./helpers/fixtures');
 const { createGasUsedTable } = require('./helpers/table');
 const { UniswapV3Pools } = require('./helpers/pools');
+const { RoutePlanner, CommandType } = require('@uniswap/universal-router-sdk');
+
+Error.stackTraceLimit = 1000;
```

Comment:
Is it required debug setting?

Reply:
I always forget to remove this, will fix.